superdesk/io/feed_parsers/ninjs.py

```diff
@@ -120,10 +120,9 @@ def _transform_from_ninjs(self, ninjs):
                     associated_item["versioncreated"] = self.datetime(associated_item["versioncreated"])
                 item["associations"][key] = deepcopy(associated_item)
 
-        renditions = ninjs.get("renditions", {})
-        if renditions.get("baseImage") or renditions.get("original"):
-            href = ninjs.get("renditions", {}).get("original", {}).get("href")
-            item["renditions"] = {"baseImage": {"href": href}, "original": {"href": href}}
+        if ninjs.get("renditions"):
+            item["renditions"] = ninjs["renditions"]
```

Comment:
I would still do some parsing there to only get relevant info and make sure it's all using correct types